index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute page | Microverse</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <div id="main">
+      <h1 id="title">Nelson Mandela's tribute page</h1>
+      <p>This is collaborative project</p>
+      <figure id="img-div">
+        <img id="image" src="https://www.un.org/en/events/mandeladay/assets/img/featured-image-tribute.jpg" alt="Nelson Mandela's tribute" width="100%">
+        <figcaption id="img-caption">
+          Nelson Mandela showed what is possible for our world and within each one of us — if we believe, dream and work together. Let us continue each day to be inspired by his lifelong example and his call to never cease working for a better and more just world. 
+        </figcaption>
+      </figure>
```

Comment:
You would like to wrap all these lines of code inside of an &lt;header&gt; element for semantic reasons.

Reply:
Nice one here!! However, Let's make sure that we always include an explanation, a "why", for each requested change. For example:

This is a well explained request: Why don't you wrap all these lines of code inside of an <header> element, that would improve the HTML structure and make the page more accessible

This request is not well explained: You would like to wrap all these lines of code inside of an <header> element for semantic reasons.